drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/GuidedDecisionTableGraphEditorServiceImpl.java

```diff
@@ -104,213 +107,205 @@ public GuidedDecisionTableGraphEditorServiceImpl( final @Named("ioStrategy") IOS
     }
 
     @Override
-    public Path copy( final Path path,
-                      final String newName,
-                      final Path targetDirectory,
-                      final String comment ) {
+    public Path copy(final Path path,
+                     final String newName,
+                     final Path targetDirectory,
+                     final String comment) {
         try {
-            return copyService.copy( path,
-                                     newName,
-                                     targetDirectory,
-                                     comment );
-
-        } catch ( Exception e ) {
-            throw ExceptionUtilities.handleException( e );
+            return copyService.copy(path,
+                                    newName,
+                                    targetDirectory,
+                                    comment);
+        } catch (Exception e) {
+            throw ExceptionUtilities.handleException(e);
         }
     }
 
     @Override
-    public List<Path> listDecisionTablesInPackage( final Path path ) {
+    public List<Path> listDecisionTablesInPackage(final Path path) {
         try {
-            final Package pkg = projectService.resolvePackage( path );
-            if ( pkg == null ) {
+            final Package pkg = projectService.resolvePackage(path);
+            if (pkg == null) {
                 return Collections.emptyList();
             }
 
             final Path pkgPath = pkg.getPackageMainResourcesPath();
-            final org.uberfire.java.nio.file.Path nioPkgPath = Paths.convert( pkgPath );
+            final org.uberfire.java.nio.file.Path nioPkgPath = Paths.convert(pkgPath);
 
-            final List<Path> paths = findDecisionTables( nioPkgPath );
+            final List<Path> paths = findDecisionTables(nioPkgPath);
             return paths;
-
-        } catch ( Exception e ) {
-            throw ExceptionUtilities.handleException( e );
+        } catch (Exception e) {
+            throw ExceptionUtilities.handleException(e);
         }
     }
 
-    private List<Path> findDecisionTables( final org.uberfire.java.nio.file.Path nioRootPath ) {
+    private List<Path> findDecisionTables(final org.uberfire.java.nio.file.Path nioRootPath) {
         final List<Path> paths = new ArrayList<>();
-        final DirectoryStream<org.uberfire.java.nio.file.Path> directoryStream = ioService.newDirectoryStream( nioRootPath );
-        for ( org.uberfire.java.nio.file.Path nioPath : directoryStream ) {
-            final Path path = Paths.convert( nioPath );
-            if ( resourceType.accept( path ) ) {
-                paths.add( path );
+        final DirectoryStream<org.uberfire.java.nio.file.Path> directoryStream = ioService.newDirectoryStream(nioRootPath);
+        for (org.uberfire.java.nio.file.Path nioPath : directoryStream) {
+            final Path path = Paths.convert(nioPath);
+            if (!dotFileFilter.accept(nioPath) && resourceType.accept(path)) {
```

Comment:
This is the fix.. to also filter "dot files" (there were other approaches, but using ```DotFileFilter``` made testing much easier).